tpot/base.py

```diff
@@ -718,8 +761,16 @@ def export(self, output_file_name):
         if self._optimized_pipeline is None:
             raise RuntimeError('A pipeline has not yet been optimized. Please call fit() first.')
 
+        to_write = export_pipeline(self._optimized_pipeline, self.operators, self._pset, self._optimized_pipeline_score)
+
+        #dont export a pipeline you just had
+        if skip_if_repeated and (self._exported_pipeline_text == to_write):
+            return False
```

Comment:
Please document the meaning of the `True`/`False` return values for the `export` function in the docstrings.